src/Repository/UserRepository.php

```diff
@@ -47,4 +50,18 @@ public function delete(User $user): void
         $em->remove($user);
         $em->flush();
     }
+
+    public function getApplicantsForActivity(Activity $activity): QueryBuilder
```

Comment:
There is already a method for this: `ActivityUserRepository::getApplicantsForActivity()`

Reply:
The method form ActivityUserRepository didn't work with paginator. So I deleted it and created a new one in UserRepository.

Reply:
https://github.com/doctrine/orm/issues/2596